### DISCUSSION TITLE
How are the intersection points obtained in OnCollision?

### DISCUSSION THREAD

**Discussion by Vasilisk8** [flame-engine/flame#2099] [Q&A] [answered]
I consider the collision of two hit boxes, sometimes there are two intersection points, sometimes three, and sometimes six, why is it so, how does Flame calculate these points?

**Comment by spydon** [comment, marked as answer]
The collisions are only calculated once per tick, this means that almost always your hitboxes will overlap a bit and every intersection point is where the hitboxes intersect at this time.
